### ISSUE TITLE
colloquial returns yesterday when it's actually not yesterday

### ISSUE THREAD

**Issue by eastsss** [malcommac/SwiftDate#397]
Hi and thx for the great library.
 But there is an issue with colloquials - sometimes it still shows "yesterday" even despite it's actually not yesterday. For example when you have two dates, where the first one is "11.04.2017 23:00" and the second one is "13.04.2017 01:00" - it will return "yesterday" because difference between the dates is more than 1 day (24 hours), but it's not actually "yesterday" anymore. I think we need some adjustments in `DateInRegionFormatter.colloquial(from:to:)`logic.

**Comment by malcommac** [comment]
Thank you for your report. Just fixed in 4.1.0.
Reference commit is https://github.com/malcommac/SwiftDate/commit/0c23cb8adba2a3c2ef3716b9df8e75533be365de

**Comment by eastsss** [comment]
it still doesn't work :( sorry, probably my explanations were bad. Please take a look at this example
`let to = Date()
let from = Date(timeIntervalSince1970: to.timeIntervalSince1970 - 47 * 60 * 60)
        
let fromDate = DateInRegion(absoluteDate: from, in: nil)
let toDate = DateInRegion(absoluteDate: to, in: fromDate.region)
let formatter = DateInRegionFormatter()
formatter.localization = Localization(locale: fromDate.region.locale)
formatter.imminentInterval = 1
let col = try! formatter.colloquial(from: fromDate, to: toDate).colloquial
        
print("col = \(col)")`

`from = 2017-04-01 15:07:52 UTC
to = 2017-04-03 14:07:52`

shows
> yesterday

so basically if difference between two dates more(!) than 24 hours, but less than 48 hours, and its not two consecutive days it still shows "yesterday"
